charts_common/lib/src/chart/common/behavior/line_point_highlighter.dart

```diff
@@ -495,7 +495,7 @@ class _LinePointLayoutView<D> extends LayoutView {
     }
 
     // Draw the highlight shapes on top of all follow lines.
```

Comment:
is this comment true now or was it true before and it's out of date now?

Reply:
From what I can tell, I think it's still true. These are the highlight dots on the vertical line? This just changes the order drawn of each dot on that line?

Reply:
Yeah, the comment is still true, just changes the order of the dots. Only workaround I could make work so that the projection dot shows on top. As a note the order of the data series collection matters when it comes to laying out lines -- seemed as if the points were being rendered in reverse. There might be a fix higher up somewhere but this did it.